client/state/test/index.js

```diff
@@ -0,0 +1,48 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		createReduxStore = require( 'state' ).createReduxStore;
```

Comment:
>If you try to move it, you'll still get:

Looks like Babel is rearranging the imports, causing `lib/react-test-env-setup` to be required _after_ `state` :frowning: 

```bash
$ ./node_modules/.bin/babel client/state/test/index.js
```

```js
'use strict';

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

/**
 * External dependencies
 */

var _chai = require('chai');

var _sinon = require('sinon');

/**
 * Internal dependencies
 */

var _sinon2 = _interopRequireDefault(_sinon);

var _state = require('state');

require('lib/react-test-env-setup')();

describe('state', function () {
	describe('createReduxStore', function () {
		it('can be called without specifying initialState', function () {
			var reduxStoreNoArgs = (0, _state.createReduxStore)().getState();
			var reduxStoreWithEmptyState = (0, _state.createReduxStore)({}).getState();
			(0, _chai.expect)(reduxStoreNoArgs).to.be.an('object');
			(0, _chai.expect)(reduxStoreWithEmptyState).to.eql(reduxStoreNoArgs);
		});
		it('is instantiated with initialState', function () {
			var user = { ID: 1234, display_name: 'test user', username: 'testuser' };
			var initialState = {
				currentUser: { id: 1234 },
				users: { items: { 1234: user } }
			};
			var reduxStoreWithCurrentUser = (0, _state.createReduxStore)(initialState).getState();
			(0, _chai.expect)(reduxStoreWithCurrentUser.currentUser).to.eql({ id: 1234 });
			(0, _chai.expect)(Object.keys(reduxStoreWithCurrentUser.users.items).length).to.eql(1);
			(0, _chai.expect)(reduxStoreWithCurrentUser.users.items[1234]).to.eql(user);
		});
		describe('invalid data', function () {
			var consoleStub;
			before(function () {
				consoleStub = _sinon2['default'].stub(console, 'error');
			});
			after(function () {
				consoleStub.restore();
			});
			it('ignores non-existent keys', function () {
				(0, _chai.expect)(consoleStub.calledOnce).to.eql(false);
				var reduxStoreNoArgs = (0, _state.createReduxStore)().getState();
				var reduxStoreBadData = (0, _state.createReduxStore)({ some: { bad: { stuff: true } } }).getState();
				(0, _chai.expect)(reduxStoreBadData).to.eql(reduxStoreNoArgs);
				(0, _chai.expect)(consoleStub.calledOnce).to.eql(true);
			});
		});
	});
});
```

Alternatively, you could use `require` to pull in `createReduxStore`, but that feels a bit ugly.